.github/release.yml

```
@@ -0,0 +1,41 @@
+# .github/release.yml
+
+changelog:
+  categories:
+    - title: Hyperion
+      labels:
+        - hyperion
+    - title: Jungfrau
+      labels:
+        - jungfrau
+    - title: Murko
+      labels:
+        - murko
+    - title: I03
+      labels:
+        - i03
+    - title: I04
+      labels:
+        - i04
```

Should: Missed out i04-1
```suggestion
    - title: I04
      labels:
        - i04
    - title: I04-1
      labels:
        - i04-1
```